dev/AppNotifications/ShellLocalization.cpp

```diff
@@ -0,0 +1,272 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include <ShellLocalization.h>
+#include <AppNotificationUtility.h>
+
+#include <cwctype>
+#include <filesystem>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/base.h>
+#include <externs.h>
+
+#include <ShObjIdl_core.h>
+#include <wincodec.h>
+#include <Ocidl.h>
+#include <windows.h>
+
+namespace std
+{
+    using namespace std::filesystem;
+}
+
+typedef enum
+{
+    VERSION1,
+    VERSION5,
+} BITMAP_VERSION;
+
+winrt::com_ptr<IWICBitmapSource> ConvertWICBitmapPixelFormat(
+    winrt::com_ptr<IWICImagingFactory> const& wicImagingFactory,
+    winrt::com_ptr<IWICBitmapSource> const& wicBitmapSource,
+    WICPixelFormatGUID guidPixelFormatSource,
+    WICBitmapDitherType bitmapDitherType)
+{
+    winrt::com_ptr<IWICFormatConverter> wicFormatConverter;
+    THROW_IF_FAILED(wicImagingFactory->CreateFormatConverter(wicFormatConverter.put()));
+
+    THROW_IF_FAILED(wicFormatConverter->Initialize(wicBitmapSource.get(), guidPixelFormatSource, bitmapDitherType, nullptr, 0.f, WICBitmapPaletteTypeCustom));
+
+    // Store the converted bitmap as ppToRenderBitmapSource
+    winrt::com_ptr<IWICBitmapSource> wicBitmapSourceConverted;
+    THROW_IF_FAILED(wicFormatConverter->QueryInterface(_uuidof(IWICBitmapSource), wicBitmapSourceConverted.put_void()));
```

Comment:
(No change required)

I _think_ this can be `return wicFormatConverter.as<IWICBitmapSource>()` ?